### PR TITLE
build: tsplus-gen and add tsPlusTypes to package.json

### PR DESCRIPTION
**Pull request by tim-smart**
TODO:

- [x] Add additional operators
- [ ] Add JSON annotations and "tsPlusTypes" to final build
- [ ] Add to CI

### CODE REVIEW

File: tsplus-gen.config.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    ["effect/io/Layer#orElse", "pipeable-operator", "|"],
+    ["effect/io/Layer#provideTo", "pipeable-operator", ">>"],
+    ["effect/io/Layer#provideToAndMerge", "pipeable-operator", ">"],
+
+    ["effect/io/Schedule#andThen", "pipeable-operator", "/"],
+    ["effect/io/Schedule#andThenEither", "pipeable-operator", "+"],
+    ["effect/io/Schedule#bothInOut", "pipeable-operator", "**"],
+    ["effect/io/Schedule#choose", "pipeable-operator", "+"],
+    ["effect/io/Schedule#compose", "pipeable-operator", ">>"],
+    ["effect/io/Schedule#composeUsing", "pipeable-operator", "<<"],
```
Comment (IMax153) [inline tsplus-gen.config.json:34]: One small comment - we no longer have `Schedule.composeUsing` in `io`.

Comment (tim-smart) [reply tsplus-gen.config.json:34]: Thanks. Also noticed the + operator is duplicated.

**Comment by tim-smart** [comment]
@IMax153 What is required to get this in the final build?

**Comment by IMax153** [comment]
Not sure - would defer to @mikearnaldi or @0x706b on that.

**Comment by mikearnaldi** [comment]
@tim-smart we'd need to integrate it in the build scripts which we are due to port independently to the https://github.com/repo-tooling/ org I think we can do in steps, I'll take a look at a minimal integration into current asap and advise further

**Pull request closed without merging**